src/System.Net.Requests/src/System/Net/_TLSstream.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net
+{
+    internal class TlsStream : NetworkStream
+    {
+        private SslStream _sslStream;
+        private string _host;
+        private X509CertificateCollection _clientCertificates;
+
+        public TlsStream(NetworkStream stream, Socket socket) : base(socket)
+        {
+            _sslStream = new SslStream(stream);
+        }
+
+        public TlsStream(NetworkStream stream, Socket socket, string host, X509CertificateCollection clientCertificates) : base(socket)
+        {
+            _sslStream = new SslStream(stream);
+            _host = host;
+            _clientCertificates = clientCertificates;
+        }
+
+        public void AuthenticateAsClient()
+        {
+            _sslStream.AuthenticateAsClient(_host, _clientCertificates, SslProtocols.Tls, false);
+        }
+
+        public IAsyncResult BeginAuthenticateAsClient(AsyncCallback asyncCallback, object state)
+        {
+            return _sslStream.BeginAuthenticateAsClient(_host, _clientCertificates, SslProtocols.Tls, false, asyncCallback, state);
```

Comment:
This is the wrong default. I'll leave comments regarding TlsStream in #12416 only.